src/components/Modals/CreateSingleItemModal/CreateSingleItemModal.tsx

```diff
@@ -834,7 +843,7 @@ export default class CreateSingleItemModal extends React.PureComponent<Props, St
   }
 
   handleOnScreenshotTaken = (screenshot: string) => {
-    this.setState({ thumbnail: screenshot, isLoading: true }, () => this.setState({ view: CreateItemView.DETAILS }))
+    this.setState({ thumbnail: screenshot }, () => this.setState({ view: CreateItemView.DETAILS }))
```

Comment:
Is this `() => this.setState({ view: CreateItemView.DETAILS })` necessary? Is not enough with the one [here](https://github.com/decentraland/builder/pull/2251/files#diff-14a508dde6f9bf5af42a88ac3c6859b51313837d30291f7c8cee96199399e024R117)?

Reply:
good catch! updating 🙌